clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -1900,6 +1900,29 @@ void Sema::ConstructOpenCLKernel(FunctionDecl *KernelCallerFunc,
   ConstructingOpenCLKernel = false;
 }
 
+// This function marks all the callees of explicit SIMD kernel
+// with !sycl_explicit_simd. We want to have different semantics
+// for functions that are called from SYCL and E-SIMD contexts.
+// Later, functions marked with !sycl_explicit_simd will be cloned
+// to maintain two different semantics.
+void Sema::MarkSyclSimd() {
+  for (Decl *D : syclDeviceDecls()) {
+    if (auto SYCLKernel = dyn_cast<FunctionDecl>(D)) {
```

Comment:
You probably don't need the braces for this 'for' and 'if'.

Reply:
Fixed.